### DISCUSSION TITLE
RSP from accelerometer data on the chest

### DISCUSSION THREAD

**Discussion by stamate** [neuropsychology/NeuroKit#924] [Ideas]
Hi all, how easy would it be to interpret accelerometer data (the device would be on the chest of a person, in suppine position, or strapped to the chest) in the current RSP framework? Should we focus on converting the data into more resp. strap like data? or just write another wrapper for it?

**Comment by DominiqueMakowski** [comment]
That's a interesting question 🤔 
I'm not sure to be honest, maybe you can try recording and plotting here an example of ACC signal from the chest to see how it looks like, and whether there would be a way to extract the rsp phases from it.

**Reply by stamate** [reply]
Ok, great, will do! Thanks!
